libvips/foreign/jxlload.c

```diff
@@ -547,7 +536,7 @@ vips_foreign_load_jxl_read_frame(VipsForeignLoadJxl *jxl, VipsImage *frame,
 		case JXL_DEC_FULL_IMAGE:
 			/* We decoded the required frame and can return
 			 */
-			if (jxl->frame_no >= frame_no)
+			if (jxl->frame_no == frame_no)
```

Comment:
`>=` feels a bit safer. What if this function is asked for an earlier frame? Without a `>=` it'll get stuck in a loop.

Reply:
Commit 4c8dde602d14171c57d466ea97c24f3348c7b166 reverts this back to `>=`.